utils/mlocate/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mlocate
+PKG_VERSION:=0.26
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://releases.pagure.org/mlocate/
+PKG_HASH:=3063df79fe198fb9618e180c54baf3105b33d88fe602ff2d8570aaf944f1263e
+
+PKG_INSTALL:=1
+PKG_FORTIFY_SOURCE:=0
```

Comment:
Ahem. Care to explain why?

I prefer that format errors be fixed and patched.

Reply:
The problem is in fortify stdlib realpath `build/staging_dir/toolchain-arm_cortex-a9+vfpv3_gcc-7.4.0_musl_eabi/include/fortify/stdlib.h:40:13: error: 'realpath' undeclared here (not in a function); did you mean 'rpl_realpath'?
 _FORTIFY_FN(realpath) char *realpath(const char *__p, char *__r)
` I will try to fix that with patch instead of disabling fortify

Reply:
There are only two packages in the repo that use this variable. I don't want another one.

If the error is actually in fortify-headers, that should be patched first.